lib/view_model/active_record/update_operation.rb

```diff
@@ -49,7 +49,7 @@ def built?
     end
 
     def association_changed!(association_name)
-      @changed_associations << association_name
+      @changed_associations << association_name.to_s
```

Comment:
Yay this fixes it, but did you find out where the discrepancy comes from? Since most of our code works, would it be clearer to use `raise unless string`?